src/levanter/models/backpack.py

```diff
@@ -241,21 +245,23 @@ def init(
             key=k_mlp,
             use_bias=config.use_bias,
         )
+        ln_2 = hnn.LayerNorm.init(config.Senses, eps=config.layer_norm_epsilon)
```

Comment:
Same here, do we need it to be `(config.Senses, config.Embed)`?

Reply:
i think just config.Embed. I think we just want to ensure that each sense's vector is normalized, which means the embedding dim should be normalized.